apps/src/blockly/themes/cdoAccessibleThemes.js

```diff
@@ -7,11 +7,33 @@ import {
   tritanopiaBlockStyles,
 } from './cdoAccessibleStyles';
 
+// Whenever cdoBlockStyles.mjs is updated, accesibility styles in cdoAccessibleStyles.js
+// must also be updated. The accessible styles are generated by running accessibleStylesGenerator.mjs.
+// However, the ai_blocks style remains the same for all themes.
+// So we replace the `ai_blocks` style generated by accessibleStylesGenerator.mjs
+// with the the hard-coded color below.
+let protanopiaBlockStylesWithSelectedAiColor = {
+  ...protanopiaBlockStyles,
+};
+protanopiaBlockStylesWithSelectedAiColor.ai_blocks = {colourPrimary: '#566065'};
```

Comment:
Should this override instead happen in `accessibleStylesGenerator.mjs` so that all the logic for determining accessible color and associated overrides is in once place? If/when we have more instances of these kinds of overrides, I can imagine it becoming confusing to figure out how a specific color gets determined.

Reply:
Thanks! I am going to do a refactor based on @mikeharv 's suggestion and this will address your concern.